packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -0,0 +1,146 @@
+
+For example:
+
+```
+HTTP/1.1 200 OK
+X-Rate-Limit-Limit: 60
+X-Rate-Limit-Remaining: 35
+X-Rate-Limit-Reset: 1516307596
+```
+
+When a specific client/IP/device identifier combination exceeds either the per minute request limit (60 requests per minute) or the concurrent limit (two concurrent requests), the `/authorize` request returns an HTTP 429 error.
+
+### How to enable this feature
+
+> **Note:** For all orgs created after September 2020, this feature is automatically set to **Enforce and log per client (recommended)** mode.
+
+For existing orgs, to configure client-based rate limiting:
+
+> **Note:** The Admin Console is required for these steps. If you see **Developer Console** in the top left of the page, click it and select **Classic UI** to switch.
+
+1. From the Admin Console select **Settings** > **Account**, and then scroll down to the **Client-based rate limiting** section.
+
+2. Select the type of **Rate limit per client** that you want to implement:
+
+    * Select **Log and enforce client (recommended)** to enable client-based rate limiting.
+
+    * Select **Log per client** to enable client-based rate limiting in preview mode.
+
+    * Select **No action** to disable client-based rate limiting.
+
+### Frequently asked questions
+
+**Question:** Which endpoints are covered under client-based rate limiting?<br>
+**Answer:** Today, client-based rate limiting only applies to an authorization server's `/authorize` endpoint.
+
+**Question:** How is the client-specific rate limit determined?<br>
+**Answer:** The client rate-limiting framework calculates the per client rate limit based on the OAuth client ID, the user's IP address, and the Okta device identifier (the Okta device identifier that Okta sets in the browser).
+
+**Question:** What happens if my network contains a proxy server through which the requests are proxied?<br>
+**Answer:** Requests would appear to come from the same IP Address. When `/authorize` requests are made from behind a proxy IP address, make sure to [configure the respective IPs as proxies](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Security_Network). This allows the client-based rate-limiting framework to look for the IP address before the proxy to find the true client IP address.
+
+**Question:** Can I update the per client rate limiting today?<br>
+**Answer:** No. Today every client ID/IP address/device identifier combination gets 60 total requests per minute and a maximum of two concurrent requests.
+
+**Question:** Does the org-wide rate limit still apply when I enable client-based rate limiting?<br>
+**Answer:** Yes. The underlying org-wide rate limiting still applies to your Okta tenant. When the cumulative total request or maximum concurrent requests from every unique client exceeds the org-wide rate limits, your Okta tenant experiences org-wide rate-limit errors.
```

Comment:
Also don't think we need to repeat the question in the answer:
"Yes. ~The underlying org-wide rate limiting still applies to your Okta tenant.~ When the cumulative total request or maximum concurrent requests from every unique client exceeds the org-wide rate limits...